pkg/apis/autoscaling/annotation_validation.go

```diff
@@ -29,6 +30,9 @@ import (
 	"knative.dev/serving/pkg/autoscaler/config/autoscalerconfig"
 )
 
+var unitPattern = regexp.MustCompile(
+	"(K|M|G|T|P|E|Ki|Mi|Gi|Ti|Pi|Ei|)")
+
```

Comment:
Maybe we can just use K8s Quantity type for validation?

https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#Quantity

Reply:
+1